config/routes.rb

```diff
@@ -2,11 +2,11 @@
 
   resources :items, only: [:index]
 
-  resources :travesties, only: [:show] do
+  resources :travesties, only: [:show], param: :slug do
     resources :items, only: [:index]
   end
 
-  get '/cart', to: 'sessions#show'
+  get '/cart', to: 'carts#show'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.